daemon/commit.go

```diff
@@ -155,7 +155,7 @@ func (daemon *Daemon) CreateImageFromContainer(ctx context.Context, name string,
 		return "", err
 	}
 
-	id, err := daemon.imageService.CommitImage(backend.CommitConfig{
+	id, err := daemon.imageService.CommitImage(ctx, backend.CommitConfig{
```

Comment:
Slightly orthogonal, but I see that `daemon.imageService.TagImage` below also doesn't get passed the context (and contains a `context.TODO()`). I guess that can be handled separately (just went looking if there's more contexts to pass through)

Reply:
`TagImage` is gone in https://github.com/moby/moby/pull/44781